media/css/firefox/whatsnew/whatsnew-lite.scss

```diff
@@ -61,72 +62,111 @@
 //* -------------------------------------------------------------------------- */
 // Page content
 
+.lite-container {
+    display: flex;
+    align-items: center;
+    margin-left: 80px;
+
+    img {
+        margin-right: $spacing-md;
+    }
+}
+
 .lite-body {
-    background: linear-gradient(to top, #13e4ff, #0c85ff);
+    background: url('/media/img/firefox/whatsnew/lite/background-swoosh.png') no-repeat bottom, linear-gradient(#00e4ff, #3aa1ff, #0484ff);
     color: #fff;
     overflow: hidden;
     position: relative;
-    text-align: center;
+    padding-top: 10%;
 }
 
 .lite-text {
     @include text-display-md;
-    border-bottom: 2px solid $color-white;
-    border-top: 2px solid $color-white;
     font-weight: bold;
     margin: 0 auto $spacing-lg;
     max-width: 20em;
     padding: $spacing-sm 0;
 }
 
 .lite-content {
-
     .lite-woman {
         display: none;
     }
 
+    .lite-copy {
+        text-align: center;
+        z-index: 1;
+    }
```

Comment:
The layout gets a bit wonky at sub-desktop sizes when balloon lady is hidden. I think we can fix a lot of that by adding `max-width: $content-sm; margin: 0 auto;` here but might then need to override that at desktop sizes.

![image](https://user-images.githubusercontent.com/205591/50879106-8ccc6500-138e-11e9-9e5e-c272e56339e8.png)